docs/content/manual/upgrade/fully-airgapped-upgrade.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Upgrade Harvester in Fully Airgapped Environment
+---
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Airgapped Network without internet connectivity
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
+1. Network have at least two NICs
+1. Suggest each Harvester node have at lease 6 core CPU, 16GB memory and 200GB disk
```

Comment:
Why 6 core CPU, 16GB memory, when we have mentioned as 8-core processor minimum for testing and 32 GB minimum in our official docs?

Reply:
I checked our official document, we did not mention the CPU, disk and memory requirement for upgrade test. 
It's more of from the past testing experience to prevent unexpected failure. 
I would remove this line of requirement in all related upgrade test plan

Reply:
I think to have a Harvester cluster up and running, we need to meet the min requirements https://docs.harvesterhci.io/v1.1#hardware-requirements, so its okay to include the minimum requirement here as well.